src/data/real/cardinality.lean

```diff
@@ -111,7 +113,78 @@ begin
     rw [←power_def, mk_bool, mk_nat], exact 1 / 3, norm_num, norm_num }
 end
 
+/-- The cardinality of the reals, as a set. -/
+lemma mk_univ_real : mk (set.univ : set ℝ) = 2 ^ omega.{0} :=
+by rw [mk_univ, mk_real]
+
+/-- The reals are not countable. -/
 lemma not_countable_real : ¬ countable (set.univ : set ℝ) :=
-by { rw [countable_iff, not_le, mk_univ, mk_real], apply cantor }
+by { rw [countable_iff, not_le, mk_univ_real], apply cantor }
+
+/-- The cardinality of the interval (a, ∞). -/
+lemma mk_real_Ioi (a : ℝ) : mk (Ioi a) = 2 ^ omega.{0} :=
+begin
+  refine le_antisymm (mk_real ▸ mk_set_le _) _,
```

Comment:
Just a tip, I don't care if you change it in this PR: 
I personally tend to always avoid `▸`; it's a weaker version of `rw`, and in some cases more brittle (although that is also me as a Lean 2 user talking; since the behavior of `▸` was a lot more unpredictable in Lean 2, and proofs could easily break with sequences of these substitutions). 
In this case I think you can do:
```
refine le_antisymm _ _, { rw mk_real, exact mk_set_le _ }
```
I don't mind if you don't change it in this PR: I don't think it's an actual style guide, and all use cases are single line proof terms in this PR.